styles.css

```diff
@@ -0,0 +1,837 @@
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .skill-card-icon {
+    position: relative;
+    top: 10%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .skill-card-icon-container {
+    width: 80px;
+    height: 80px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 100%;
+  }
+
+  .skill-card-content {
+    width: 91%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+    position: relative;
+    bottom: -100px;
+  }
+
+  .skill-card-title {
+    width: 100%;
+    height: 32px;
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+  }
+
+  .skill-card-tech {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    width: 100%;
+  }
+
+  .skill-card-rows {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    width: 100%;
+  }
+
+  .skill-card-tech-tag {
+    display: flex;
+    align-items: center;
+    background: #fff;
+    border-radius: 8px;
+    height: 40px;
+    justify-content: center;
+  }
+
+  .skill-card-tech-tag-text {
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #36b37e;
+  }
+
+  .ljavascript {
+    width: 92px;
+  }
+
+  .lruby {
+    width: 55px;
+  }
+
+  .lhtml {
+    width: 53px;
+  }
+
+  .lcss {
+    width: 50px;
+  }
+
+  .bootstrap {
+    width: 84px;
+  }
+
+  .rails {
+    width: 103px;
+  }
+
+  .rspec {
+    width: 63px;
+  }
+
+  .capybara {
+    width: 83px;
+  }
+
+  .selenium {
+    width: 81px;
+  }
+
+  .codekit {
+    width: 72px;
+  }
+
+  .github {
+    width: 67px;
+  }
+
+  .codepen {
+    width: 79px;
+  }
+
+  .gitlab {
+    width: 61px;
+  }
+
+  .terminal {
+    width: 77px;
+  }
+
+  #form {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 64px 24px;
+    gap: 24px;
+  }
+
+  #form-text {
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    padding: 80px 0;
+  }
+
+  #form-text-item {
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+    color: #172b4d;
+    text-align: center;
+  }
+
+  #form-content {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    align-items: centre;
+  }
+
+  .form-row {
+    width: 100%;
+    height: 48px;
+    display: flex;
+    gap: 24px;
+    padding: 7px;
+  }
+
+  #first-form-row {
+    display: none;
+  }
+
+  .form-row-column {
+    width: 50%;
+    padding: 7px;
+  }
+
+  .textareas {
+    height: 114px;
+    width: 100%;
+    padding: 6px;
+  }
```

Comment:
- [ ] [OPTIONAL] Please make the color of the  `outline` `#36B37E` instead of default `blue`  when the input field is put in focus, as mentioned in the Figma design. This will make your site look cool 😎 and will emphasize your attention to detail 😄  

Figma:

<img width="431" alt="Screenshot 2023-02-22 at 11 25 59 AM" src="https://user-images.githubusercontent.com/44798044/220542518-628e009c-a34a-41e1-81fa-2453afe5f048.png">

Yours:

<img width="727" alt="Screenshot 2023-02-22 at 11 25 51 AM" src="https://user-images.githubusercontent.com/44798044/220542549-6c7c06a0-465f-4484-b058-8b18c21fe24b.png">